back/sssscs/src/main/java/com/ib/certificate/Certificate.java

```diff
@@ -55,9 +54,16 @@ public enum Type {
 	@Enumerated(value = EnumType.STRING)
 	private Type type;
 	
+	//@Column(nullable = false)
+	private String serialNumber;
+	
 	public Certificate(CertificateRequest req) {
-		setIssuer(req.getIssuer());
 		setParent(req.getParent());
+		if (getParent() != null) {
+			setIssuer(req.getParent().getIssuer());
+		} else {
+			setIssuer(null);
+		}
```

Comment:
Yes, this was an oversight because we don't need to store "issuers". Fixed in [e0e6910](https://github.com/magley/sssscs/pull/5/commits/e0e691099ce41198adfd10b1fb405b75e63cf07b).